.github/workflows/codeql.yml

```diff
@@ -2,6 +2,7 @@ name: "CodeQL"
 
 on:
   push:
+    branches: [ 'master' ]
```

Comment:
IMHO don't add this, i think this is a rebase / merge artifact.

Reply:
it is, got confused because of the first reply to this pr